CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ### Bugfixes
 
+* Lorem ipsum.
+
+### Breaking changes
+
+* Lorem ipsum.
+
+### Enhancements
+
+* Lorem ipsum.
+
+### Internals
+
+* Implemented inter-process CondVars on Windows (Win32 + UWP). They should be fair and
+  robust.
+
+----------------------------------------------
+# 2.6.2 Release notes
+
+### Bugfixes
+
```

Comment:
This _will_ change (and cause conflicts) after 2.6.2 has been released, so please wait a bit.